grails-app/views/manageLayers/layer.gsp

```diff
@@ -78,30 +79,35 @@
                             <th>Name</th>
                             <th>Description</th>
                             <th>sname</th>
-                            <th></th>
-                            <th></th>
+                            <th colspan="3"></th>
                             </thead>
                             <tbody>
                             <g:each in="${fields}" var="item">
-                                <tr>
-                                    <td>${item.id}</td>
-                                    <td>${item.name}</td>
-                                    <td>${item.desc}</td>
-                                    <td>${item.sname}</td>
+                                <g:set var="disabledField" value="${item.enabled ? '' :'disabled'}" />
+                                <tr >
+                                    <td class="${disabledField}" >${item.id} <g:if test="${disabledField}"> ( ${disabledField} ) </g:if></td>
+                                    <td class="${disabledField}">${item.name}</td>
+                                    <td class="${disabledField}">${item.desc}</td>
+                                    <td class="${disabledField}">${item.sname}</td>
+
                                     <td><g:link controller="manageLayers" action="field" id="${item.id}">
                                         <i class="glyphicon glyphicon-edit"></i>
                                         edit
-                                    </g:link>
+                                      </g:link>
                                     </td>
                                     <td><g:link controller="object" action="fieldObjects" id="${item.id}">
                                         <i class="glyphicon glyphicon-list"></i>
                                         list objects
-                                    </g:link>
+                                       </g:link>
                                     </td>
-                                    <td><a onclick="return confirmDelete('${item.id}');">
-                                        <i class="glyphicon glyphicon-remove"></i>
-                                        delete
-                                    </a>
+                                    <td>
+                                        <g:if test="${item.enabled}">
+                                            <a onclick="return confirmDelete('${item.id}','${item.name}' );">
+                                                <i class="glyphicon glyphicon-remove"></i> </a>
+                                        </g:if>
+                                        <g:else>
+                                            <i class="glyphicon glyphicon-ban-circle" data-toggle="tooltip" data-placement="top" title="You are not allowed to delete a disabled field."></i>
```

Comment:
Same as above. Remove this unnecessary logic. Deleting a disabled item will not break anything.